extensions/amp-subscriptions-google/0.1/amp-subscriptions-google.js

```diff
@@ -478,6 +478,31 @@ export class GoogleSubscriptionsPlatform {
    */
   pingback() {}
 
+  /** @override */
+  completeDeferredAccountCreation(entitlements) {
+    return this.runtime_
+      .completeDeferredAccountCreation({
+        entitlements,
+      })
+      .then(
+        () => {
+          // The user authorized the creation of a linked account.
+          // Complete all the logging and then...
+          return this.runtime_
```

Comment:
Very much ideally this should be happening inside the SwG library. I'm ok to allow this but IMHO, SwG should be updated to do this in the meantime.